pulsar-broker/src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java

```diff
@@ -79,16 +80,7 @@
 import org.apache.pulsar.common.api.proto.CommandGetTopicsOfNamespace.Mode;
 import org.apache.pulsar.common.lookup.GetTopicsResult;
 import org.apache.pulsar.common.lookup.data.LookupData;
-import org.apache.pulsar.common.naming.BundleSplitOption;
-import org.apache.pulsar.common.naming.FlowOrQpsEquallyDivideBundleSplitOption;
-import org.apache.pulsar.common.naming.NamespaceBundle;
-import org.apache.pulsar.common.naming.NamespaceBundleFactory;
-import org.apache.pulsar.common.naming.NamespaceBundleSplitAlgorithm;
-import org.apache.pulsar.common.naming.NamespaceBundles;
-import org.apache.pulsar.common.naming.NamespaceName;
-import org.apache.pulsar.common.naming.ServiceUnitId;
-import org.apache.pulsar.common.naming.TopicDomain;
-import org.apache.pulsar.common.naming.TopicName;
+import org.apache.pulsar.common.naming.*;
```

Comment:
Is this using Pulsar code style rules?

Reply:
https://github.com/apache/pulsar/actions/runs/6710630191/job/18242486566#step:8:2522
```
[INFO] There are 3 errors reported by Checkstyle 8.37 with /home/runner/work/pulsar/pulsar/buildtools/src/main/resources/pulsar/checkstyle.xml ruleset.
Error:  src/main/java/org/apache/pulsar/broker/namespace/OwnershipCache.java:[49,1] (imports) ImportOrder: Import javax.annotation.Nonnull appears after other imports that it should precede
Error:  src/main/java/org/apache/pulsar/broker/namespace/NamespaceService.java:[83,39] (imports) AvoidStarImport: Using the '.*' form of import should be avoided - org.apache.pulsar.common.naming.*.
Error:  src/main/java/org/apache/pulsar/broker/service/SystemTopicBasedTopicPoliciesService.java:[38,8] (imports) UnusedImports: Unused import: org.apache.pulsar.broker.namespace.NamespaceBundleOwnershipListener.
```

Reply:
Oh, sorry. This PR still needs change.